### ISSUE TITLE
header not touch

### ISSUE THREAD

**Issue by songxiaoliang** [JungHsuan/react-native-collapsible-tabview#7]
header not touch, i need scroll to touch header

**Comment by JungHsuan** [comment]
hi @songxiaoliang, Do you mean header is not scrollable?
if so, I have a version of scrollable header, but the performance is not quite well on Android.

**Comment by songxiaoliang** [comment]
@JungHsuan Yes, a large part of the area above my TabBar is the Header, but when the area is large, the height of the header may be larger than the mobile phone screen in extreme cases. At this time, there is no way to touch and slide, because there is no way in the heap area. Trigger sliding operation

**Comment by JungHsuan** [comment]
OK, I will create a new branch for a scrollable-header solution.

**Comment by songxiaoliang** [comment]
@JungHsuan That's really great, I look forward to it, thank you

**Comment by songxiaoliang** [comment]
@JungHsuan Hello, is there any new progress

**Comment by JungHsuan** [comment]
@songxiaoliang 
hi, sorry for my late update. 
I simply wrote an example of scrollable header. 
Please check the file ``headerScrollableDemo.js``.

**Comment by songxiaoliang** [comment]
@JungHsuan Great, very happy and thankful, I go to practice, communicate at any time, thank you again

**Comment by sanbat** [comment]
How to use ScrollView instead of FlatList in headerScrollableDemo.js

**Comment by JungHsuan** [comment]
@sanbat Hi, you need to replace ``scrollToOffset`` to ``scrollTo``.

**Comment by sanbat** [comment]
@JungHsuan Thanks a lot.. Its working

**Comment by kiwina** [comment]
Nice example! like to point out that the content over scrolls below the tab bar. you cannot notice it when the list is large but if you change it to say 4 elements, you will see it
